### DISCUSSION TITLE
Stopped recognizing videos

### DISCUSSION THREAD

**Discussion by Levael** [aclap-dev/video-downloadhelper#999] [Help and Q&A]
### Your question

Stopped recognizing videos after the update. I used to download videos a lot from that site, now all I get is 'No media to process in the current tab'. Can't provide website because you need to have a paid subscription to it

### Addon & CoApp version and details

version: 9.0.2.12
target: google
channel: stable
lang: en-US
coapp: {"found":true,"new_version":false,"path":"C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe","version":"2.0.19"}
license: {"accepted":true,"email":"<removed>","key":"06facb5f740148d7"}
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/126.0.0.0 Safari/537.36
{
    "valid_license_message_has_been_displayed": true,
    "http_media_download_strategy": "coapp",
    "used_sidebar_button": true,
    "used_popup_button": true,
    "download_directory": "C:\\Users\\aa\\dwhelper",
    "theme": "dark",
    "view_options": {
        "all_tabs": false,
        "hide_downloaded": false,
        "low_quality": true,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "sort_by_status": true,
        "sort_reverse": false
    },
    "use_legacy_ui": false,
    "use_sidebar": true,
    "open_count_store": 1
}

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I am just another user.

When you fill in the `Addon & CoApp version and details` section of a problem report, it is much easier to just follow the instructions here:

https://github.com/aclap-dev/video-downloadhelper/wiki/Where-can-I-find-my-versions%3F

Just click the `Copy information to clipboard` button, then come in here & do Shift+Insert in that section of your problem report.  You can edit your problem report.  In fact, you can edit any of your own posts.  Note the 3 dots above & to the right of any of your posts, including your first post in this discussion.  Click the 3 dots.  I think you'll be able to take it from there.

Also, VDH for Chrome has never been able to download from YouTube.  Google owns both the Chrome browser & YouTube.  It is a condition imposed by Google on any software author that in order to be offered on the Chrome web store, your software must not provide functionality to download from YouTube.  So VDH for Chrome does not download from YouTube.

If you have some other site on which you are having trouble with VDH for Chrome, you need to post a few sample URLs.  Those would be URLs of web pages on which problematic videos reside.  Don't just say `site X is a problem` & don't just say `all sites are a problem`.  Post specific URLs.  You need to provide enough information for somebody else to try the same downloads & see if the problem also occurs for the other person.  That's the scientific method.  Results need to be replicated by multiple people.  So you need to provide enough information for others to replicate your results.

However, if your sample URLs require someone to register for the site, for a fee or free, then we'll need to investigate alternative methods of trying to help you.

**Comment by Levael** [comment]
Thanks for the detailed explanation! I really should correct what you have
indicated. But really, the site is only for registered users, which is why
I didn't leave a link to it from the very beginning

пт, 12 июл. 2024 г., 03:33 GrampaWildWilly ***@***.***>:

**Comment by margo-mv** [comment]
my VDH is not working.
The service has stopped working since yesterday. I used to download any videos without any problems. Now, when starting the video, the service writes: "The current tab is empty. Start playing the video to detect it."
For example, on this site https://getcourse.ru there is a "Video about the platform" button. When launching this video, the situation is similar - the service does not see it and cannot download it. Although I downloaded an excellent one yesterday.
Help please

**Reply by margo-mv** [reply]
I use this app in Google Chrome

**Reply by Levael** [reply]
I can confirm, just checked. For the sake of experiment, I went to YouTube - the first video that comes across is perfectly recognized

**Reply by Mattmjs** [reply]
@margo-mv 
There was already a fix in the beta extension for Firefox and it should have the fix coming to chrome.

**Reply by GrampaWildWilly** [reply]
Yes, there is another discussion in here that mentions getcourse.ru.  You should search this forum for it.  Use the search field at the top of this page.

@Levael Then we need to collect the Hit Details from one of your attempted downloads.  First, open a post in this discussion so that you have an open text entry box ready for use.  Don't type anything in it yet.  You're just getting it ready.  Then in another browser window, visit one of your problematic videos & do this:

![#01](https://github.com/user-attachments/assets/6b575dd4-2d4b-4606-b914-96d879c5af98)
![#02](https://github.com/user-attachments/assets/6aab7609-26ba-49a2-9eff-c2f9ca6762d8)

That will take you to a page of useful information: the Hit Details.  On that page, execute these 2 key sequences:

- `Ctrl+a` to highlight the entire Hit Details
- `Ctrl+Insert` to copy all the highlighted text to the system clipboard

Then immediately swap windows back to this thread where your post is waiting.  In the text entry box, execute this key sequence:

- `Shift+Insert` to paste the Hit Details in here.

That's it.  Post it.

**Reply by GrampaWildWilly** [reply]
I find that when I have this VDH setting enabled

> "low_quality": true,

I get a lot of junk in my VDH menu.  It's just distracting noise.  I think that setting is there only for certain situations in which Paul might be trying to debug something & he might ask you to turn it on.  I think you would simplify your life if you turned that setting off.

**Reply by Levael** [reply]
1) I can't provide any details because the list of recognized videos is empty. Not a single one.
2) The option about poor quality is enabled because I tried to experiment and somehow make the program work. Unsuccessfully

**Reply by GrampaWildWilly** [reply]
Not encouraging.  OK.  If I were encountering this, I would open the Developer Tools of the browser & see what was coming in in the Network tab.  That would be first in an empty browser window, Ctrl+Shift+i.  Put it on the Network tab.  Then navigate to your web page.  Then click on the column header labelled Type.  That sorts the entries on their type, which groups them by their type.  Then you can save what's in there to a file that you can share here.

![#01](https://github.com/user-attachments/assets/d2ecd9a7-9c5e-4e4c-b9d9-8065ef6e7f43)
![#02](https://github.com/user-attachments/assets/c24ca348-ade8-42ca-8a02-311fa4fbb3fc)

The save operation gives you a file selection dialog so you can choose where to save the file.  I don't know how big it will be.  You might need to zip it before you post it here.  You probably should zip it because GitHub probably doesn't accept files named .har but it does take .zip.  Plus it may be bigger than GitHub will allow in a single post, even after you zip it.  In that case split the file.  It's easy to do because it's a plain text file.  So use your text editor to chop the file into however many pieces it takes so that a zip of each part can be posted in separate posts here.

Once you share that file(s), I can put it/them on my system & use another entry on that menu, the one that says `Import HAR file`.  That will give me an approximation of sitting beside you in front of your monitor & browsing your site with you.  Maybe I'll see something.  Meanwhile, maybe @Mattmjs will look at it, too.  Maybe he'll see something.  The more the merrier.  In fact, after you open the Network Monitor to gather this information, you might be inspired to sift through it yourself.  Maybe something will jump out at you.

**Reply by Levael** [reply]
Sorry for late response, temporary lost access to the site. Here it is, an example without need to register on the site:
https://vsemart.getcourse.ru/volume_paintings

There are 2 videos, one from Youtube on top, and one from that platform on the bottom, which duration is 39:59
![image](https://github.com/user-attachments/assets/ad93629a-caf8-47ab-ab5f-eb22ca1ee806)

**Reply by Mattmjs** [reply]
Your on chrome, there was already a fix for the beta extension on Firefox.

**Reply by GrampaWildWilly** [reply]
I do not work for VDH.  I'm just another user.

I'm running Windows 11 Pro 64-bit version 23H2 build 22631.3958, Firefox 128.0.3 64-bit, licensed VDH 9.0.2.14a1 beta, CoApp 2.0.19.

Right.  There were 3 items recognized by VDH on that page.  By the way, the true URL is just

https://vsemart.getcourse.ru/volume_paintings

Everything after the first ? is just trackers & you should not bother posting such superfluous parts of URLs.

I downloaded all 3 items & they all played just fine in VLC.  As Matt said, beta VDH for Firefox has no problems with this.  You might try the beta VDH for Chrome to see if it makes any difference.

**Reply by Levael** [reply]
Yes, the beta version is working. Thanks for the help and advices! Many thanks!
